packages/launcher/lib/browsers.ts

```diff
@@ -28,6 +28,22 @@ export const browsers: Browser[] = [
     profile: true,
     binary: 'google-chrome-canary',
   },
+  {
+    name: 'edgeCanary',
+    family: 'chrome',
+    displayName: 'Edge Canary',
```

Comment:
Could you also add Edge Dev & Beta? Contrary to Chrome, they can all be installed in parallel on both Windows & macOS with separate profiles.

Reply:
Let's land this change in 4.0 and then add Edge Dev and Beta separately, there are lots of changes going in, unfortunately ...

Reply:
Why do you have EdgeDev in the browser_spec? https://github.com/cypress-io/cypress/pull/6181/files#diff-c15b51d04b162559d1996b46f9c5b209

I agree that these should just be pushed into this PR. What's the reason to wait it's just a few extra fields? I need to consider these values when generating the icons because we need the exact names to pull against so I've already added them in there for testing.

Reply:
I added detection for Windows + a best guess at detection for macOS in 
ca0d69f. Verified working on Windows:

![image](https://user-images.githubusercontent.com/1151760/73571611-fc4f3f80-443c-11ea-9538-e0bcd48bb4e0.png)

Since I'm on Linux, can someone with a Mac verify that it detects Beta and Dev correctly? You can get the downloads here: https://www.microsoftedgeinsider.com/en-us/download?platform=macos

Reply:
I'll try it out on MacOS.

Reply:
Looks good on MacOS!

![Screen Shot 2020-01-31 at 3 54 39 PM](https://user-images.githubusercontent.com/1157043/73573719-13dcf700-4442-11ea-975c-b21aca17fb87.png)

I moved Edge (stable) up so it appears before the other variants.

Reply:
> I moved Edge (stable) up so it appears before the other variants.

The order is a bit weird, though - I'd expect it to be: stable, beta, dev & canary, so that they're sorted by stability & the planned release date. Having: stable, canary, beta, dev seems quite random.